react/DynamicIframe.tsx

```diff
@@ -14,14 +14,14 @@ const DynamicIframe: StorefrontFunctionComponent<DynamicIframeProps> = ({
   let allParamsExist = true
   const src = dynamicSrc.replace(/({[A-z0-9]*})/g, function(match: string) {
     const thisParam = match.replace(/{|}/g, '')
-    if (thisParam && params[thisParam]) {
-      return params[thisParam]
+    if (!thisParam && !params[thisParam]) {
```

Comment:
I think it should be `or` here, right?

```suggestion
    if (!thisParam || !params[thisParam]) {
```

Reply:
I think it should be and. we need to make sure thisParam isn't null after the match replace and that the value exists in the params array before proceeding.

Reply:
Considering this case:

thisParam = 'fooBar'
params = {}

```ts
// if (!thisParam && !params[thisParam]) 
if (false && true) { // it would be false and not enter the condition

}
```

So in this case it should be:

```ts
// if (!thisParam || thisParam && !params[thisParam])
if (false || true && true) { // it would be true

}
```

Reply:
But if either of them are null, it's enough to trigger the error condition, right?

Reply:
I don't think the `thisParam &&` is needed because of short-circuiting, right? The condition after the `||` is only evaluated if `thisParam` is not null.

Reply:
True